### ISSUE TITLE
Improve site design

### ISSUE THREAD

**Issue by janosh** [janosh/awesome-sveltekit#23]
I ain't no web designer. The current site https://awesome-svelte-kit.netlify.app looks rather rough-hewn. If any CSS wiz cares to improve the site's appearance, it would be much appreciated! 🙇‍♂️

**Comment by theonikolai** [comment]
Hey @janosh, would be more than happy to help. There are two things we should definitely consider before starting -

1st - Do you have any particular inspirations for what you'd want the site to look like.
2nd - Would we be looking at using any particular CSS frameworks?

Personally, I'm a bit of a sucker for TailwindCSS, but I'd be more than happy to adapt if needed.

**Comment by janosh** [comment]
@theonikolai Thanks for offering to help! Good questions too.

1. I think the [Gatsby showcase](https://gatsbyjs.com/showcase) looks quite nice. Perhaps we can use that as a guide. Let me know if you have other ideas. I'm definitely open to sugggestions.
2. I'd prefer for the site to remain pure CSS, i.e. no frameworks. Hopefully this is not a deal breaker for you?

**Comment by theonikolai** [comment]
Hey, if people can rave about vanilla JavaScript, what's not to love about a little Vanilla CSS?

Anyway, I'll see what I can do about this a little later once I have the time.

This would be my first time contributing though so I'm hoping things go well 🙏

**Comment by theonikolai** [comment]
Hey @janosh - Thought of starting work on the website redesign when I noticed that site/src/sites.ts was included in [.gitignore](https://github.com/janosh/awesome-svelte-kit/blob/main/.gitignore). Was this intentional?

I tried using `yarn predev`, `yarn prebuild`, and `node ./scripts/parseSiteYaml.js` from the site folder, but I can't seem to generate sites.ts

I'm not sure if I can start work on the redesign without it. Is there any way to work around this?
